### ISSUE TITLE
Authentication issue against admin GUI

### ISSUE THREAD

**Issue by afterthought** [couchbase/kubernetes#17]
I'm guessing I don't have things configured exactly right. I have the version of the service spec when the type was NodePort. I setup a test cluster last week and things seemed OK. I setup a new test cluster and now when I connect using the 30891 Nodeport I get the admin login screen and can successfully login. However, after a second or two, XHR calls fail with 401 responses and I am logged out. Does this sound like something obvious that others have already dealt with in regard to configuration for the GUI and auth?

**Comment by bcbroussard** [comment]
@afterthought the logout issue is probably caused by the service alternating requests between the 2 pods. In that case, you can try scaling down to 1 pod, making admin changes, and then scaling back up. 

```
kubectl scale --replicas=1 rc couchbase-controller
#access web ui
kubectl scale --replicas=2 rc couchbase-controller
```
@tleyden and I are discussing the best approach to take for solving this.

**Comment by afterthought** [comment]
I just confirmed the fix in #22 for the admin load balancer issue on AWS. Thanks @bcbroussard .
